Cargo.toml

```diff
@@ -13,3 +13,9 @@ members = [
 [profile.release]
 lto = true
 strip = true
+
+[profile.test]
+opt-level=1
+lto = false
+debug-assertions=true
+debug=true
```

Comment:
I don't think this should be part of this PR. Rather, it should be landed separately, so people can git blame it and revert/change it if necessary in future.